docs/source/60_building_apps.md

```diff
@@ -256,6 +256,39 @@ $ spark-submit \
 A complete example of this pattern can be found in the
 [heuermh/adam-examples](https://github.com/heuermh/adam-examples) repository.
 
+### Using ADAM’s ShuffleRegionJoin API
+Another useful API implemented in ADAM is the ShuffleRegionJoin API, which joins two genomic datasets that contain overlapping regions. This primitive is useful for a number of applications including Variant calls and Feature mapping.
```

Comment:
1. It'd be good to give a more detailed example. E.g., "The region join primitive is useful for a number of applications including variant calling (identifying all of the reads that overlap a candidate variant) or Feature mapping (what is feature mapping?)."
2. Break at 80 chars. ;)